### DISCUSSION TITLE
Separating the type of media data

### DISCUSSION THREAD

**Discussion by cgeheim** [immich-app/immich#12466] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Insert buttons on the left sidebar to display videos or images separately.

Button All
Button: Images
Button: Videos

I like the option of separating the type of media data in the view

![image](https://github.com/user-attachments/assets/04fec094-7eac-48ff-a850-1668f21526ae)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Dupe: #12238
